iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/posix_call.inl

```diff
@@ -86,7 +48,7 @@ inline PosixCallBuilder<ReturnType, FunctionArguments...>::PosixCallBuilder(Func
                                                                             const int32_t line,
                                                                             const char* callingFunction) noexcept
     : m_posixCall{posixCall}
-    , m_details{posixFunctionName, file, line, callingFunction, true, {}}
+    , m_details{posixFunctionName, file, callingFunction, line, true, false, false, {}}
```

Comment:
I am remembering the wisdom of a very old and wise man (called King Bob). He spread the word that enums are much more readable then booleans. Here we do not have a clue what the `true` and `false`s are standing for.

Would you please so kind use enum classes here instead of booleans.

Reply:
No, not in this case. It's only used for the ctor and enums make the code worse since it's not possible anymore to have a `hasIgnoredErrno |= doesContainValue(...);`. I could add a ctor to `PosixCallDetails` to set only the non default values

Reply:
done